docs/semgrep-assistant/getting-started.md

```diff
@@ -0,0 +1,129 @@
+# Enable Semgrep Assistant
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+This article walks you through enabling Semgrep Assistant for your deployment.
+
+:::info Prerequisites
+* You have completed a [Semgrep core deployment](/deployment/core-deployment).
+* You have set rules to **Comment** or **Block** mode in your [<i class="fas fa-external-link fa-sm"></i> Policies page](https://semgrep.dev/orgs/-/policies).
+:::
+
+<Tabs
+    defaultValue="github"
+    values={[
+    {label: 'GitHub', value: 'github'},
+    {label: 'GitLab', value: 'gitlab'},
+    ]}
+>
+
+<TabItem value='github'>
+
+Semgrep Assistant extends normal Semgrep capabilities by providing contextually-aware AI-generated suggestions. In order to build that context, it requires GitHub permissions in excess of the 
```

Comment:
```suggestion
Semgrep Assistant extends normal Semgrep capabilities by providing contextually aware AI-generated suggestions. In order to build that context, it requires GitHub permissions in excess of the 
```
Vale.sh says:

![Snag_29dd7cc](https://github.com/semgrep/semgrep-docs/assets/20766840/64d80940-8e85-4cec-9f46-afa3d1c5d382)

I think it's correct.

Reply:
'Tis; I always forget that words ending in `-ly` don't need hyphens.